src/encryptedData.test.ts

```diff
@@ -58,15 +62,21 @@ describe('Encrypted data API', () => {
     const serializedEncryptedData = encryptedData.serialize()
     assert.notEqual(serializedEncryptedData, 'foo')
 
-    const incoming = encryptedIncomingData('', {
-      _vm: {
-        authorizedKeys: {
-          [id]: {
-            purpose: ['enc']
+    const incoming = encryptedIncomingData(
+      '',
+      {
+        _vm: {
+          authorizedKeys: {
+            [id]: {
+              purpose: ['enc']
+            }
           }
         }
-      }
-    } as ChelContractState, serializedEncryptedData, 0, { [id]: key })
+      } as ChelContractState,
+      serializedEncryptedData,
+      0,
+      { [id]: key }
+    )
```

Comment:
Well, I suppose these types of changes aren't really needed either.

Reply:
No big deal either way I suppose. But note that prettier doesn't need to be run again.. And really we don't need to add it as a project dependency

Reply:
Indeed you're right 👍